### ISSUE TITLE
Would it be possible to implement an audio Cue to inform that a command being executed in the terminal has finished?

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#178989]
Type: <b>Feature Request</b>

When I execute a command in a terminal opened in VSCode and the command ends with a return code other than 0, an audio Cue is played to inform this fact. Honestly I find this feature very useful.

In my view it would also be useful if there was a feature that could indicate that a command being executed in the terminal has ended. I could run for example the application build that takes a long time and while the build is running I could have a coffee or a beer, or even edit some files in VSCode. As soon as the build finished I would be alerted by VSCode.
I think it would be an interesting feature for people who use a screen reader like me.

VS Code version: Code - Insiders 1.78.0-insider (2da5a0024fe2ca407f741f19340f1a10f4d692af, 2023-04-03T05:26:01.718Z)
OS version: Linux x64 6.2.8-200.fc37.x86_64
Modes:
Sandboxed: Yes

**Comment by jvesouza** [comment]
cc: @meganrogge @isidorn 

Something I miss in VScode when I'm using the terminal is the warning that a command from that terminal has ended.
While a long command is being executed I could have a coffee or a beer, or even take advantage of the waiting time to edit some files. When the command finished I would be informed by VSCode.

We have already implemented 'Audio Cues: Terminal Command Failed'. How about something like 'Audio Cues: Terminal Command Finished'?
I think it would be an interesting feature for people who use a screen reader like me.

**Comment by isidorn** [comment]
Thanks for opening this @jvesouza 
Overall I think it makes sense. How does this overlap with the Task Finished audio cues? Should it be the same?

**Comment by jvesouza** [comment]
In my opinion it should not be related to Task Finished audio cues.

**Comment by isidorn** [comment]
@jvesouza can you elaborate why do you think so :)

**Comment by jvesouza** [comment]
> > for example the application build that takes a long time and while the build is running
> 
> is there a reason you don't use a task for this?

Yes, lack of knowledge on my part. I didn't know that it is possible to use a task to run any process.
Less beer and more reading!

**Comment by ClaudMor** [comment]
Hello,

It could anyway be that the OP's request is useful to many.

For example, if someone's running a code selection in the terminal (extensions such as [Julia](https://marketplace.visualstudio.com/items?itemName=julialang.language-julia) allow this) and the code takes long to execute (which is usually the case in data science applications), being able to multi-task while the selection is running knowing that an audio cue will play when it finishes (whether it succedes or not) could be useful.

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_smiling_face: This feature request received a sufficient number of community upvotes and we moved it to our backlog. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
